src/backend/base/langflow/components/composio/composio_api.py

```diff
@@ -30,16 +30,16 @@ class ComposioAPIComponent(LCToolComponent):
             name="api_key",
             display_name="Composio API Key",
             required=True,
-            # refresh_button=True,
```

Comment:
You should add `real_time_refresh=True` here.

Reply:
We've updated the changes.